common/client-core/src/cli_helpers/client_add_gateway.rs

```diff
@@ -168,7 +167,6 @@ where
         identity: gateway_details.gateway_id,
         active: common_args.set_active,
         typ: gateway_registration.details.typ().to_string(),
-        endpoint: Some(gateway_details.published_data.listeners.primary.clone()),
-        fallback_endpoint: gateway_details.published_data.listeners.fallback.clone(),
+        endpoint: Some(gateway_details.published_data.details.clone()),
```

Comment:
that's a nit, but I think it'd make more sense to rename that field. it's no longer just an `endpoint`